lib/smack/Naming.cpp

```diff
@@ -62,6 +62,8 @@ const std::string Naming::REC_MEM_OP = "boogie_si_record_mop";
 const std::string Naming::MEM_OP_VAL = "$MOP";
 
 const std::string Naming::RUST_ENTRY = "_ZN3std2rt10lang_start";
+const std::string Naming::RUST_LANG_START_INTERNAL =
```

Comment:
Why do you need both `RUST_ENTRY` and `RUST_LANG_START_INTERNAL`?

Reply:
`RUST_ENTRY` is used for the function that appears in `@main` that calls the real main function. `RUST_LANG_START_INTERNAL` is a function which appears in the set up of closure functions.